### ISSUE TITLE
Push to Pypi

### ISSUE THREAD

**Issue by barsa2000** [excludarr/excludarr#19]
To publish excludarr on pypi we either have to rename the project or publish it under another name. (unless we already have access to the original repo but I don't think that's the case)

Thoughts?

TBH I don't ever really think that publishing on pypi is necessary but maybe someone could like to have it on there + the original was there so idk what's the best option

**Comment by TheArturthur** [comment]
Maybe we could name it something like `py-excludarr`? I don't know.
For it to have the same name we would need @haijeploeg either to set up a token, or release the name for us to use

**Comment by barsa2000** [comment]
Tbh I'm more keen on renaming it to something like "xcludarr", we could even base the logo on the X itself, or maybe "excludarr-reborn". But idk. Changing to something like "py-excludarr" to me feels like we are creating a library to do something, so I'm not sure. 

We could even ditch the idea of publishing on there too, like what's the purpose anyway? I don't think there would be many people pulling it from there anyway, most people either run the docker or run it from source to change stuff. 

I think the best option is to just ignore the problem for now and see if someone asks for excludarr to be published there and deal with it later

**Comment by scott-es** [comment]
agree, I dont think there is a practical benefit to publishing it to pypi, and can redirect efforts to improving on the app itself.

so then I'm guessing we can repurpose this https://github.com/excludarr/excludarr/blob/main/.github/workflows/release.yml 

to automate tagging of releases to github?! we might has well automatically push a new container every time we merge to main and checks pass (once we are comfortable if testing is good enough)

**Comment by barsa2000** [comment]
Yea, we could, I'm not a big fan of bumping version at every merge to main, but it's not a big deal.

I personally prefer to manually tag PRs because we could need more than one PR for each feature so it wouldn't be that useful to bump version at each merge IMHO. 

But it's more of a personal preference so whatever, we should at least make a development branch then, so we merge to development first and when a feature is stabilized we merge to main without removing the branch

**Comment by scott-es** [comment]
> Yea, we could, I'm not a big fan of bumping version at every merge to main, but it's not a big deal.
> 
> I personally prefer to manually tag PRs because we could need more than one PR for each feature so it wouldn't be that useful to bump version at each merge IMHO.
> 
> But it's more of a personal preference so whatever, we should at least make a development branch then, so we merge to development first and when a feature is stabilized we merge to main without removing the branch

I'm fine with the way it is, time can be better spent elsewhere.  Point taken on needing a dev branch, keeping it as simple as required sounds good.
